src/main/java/io/vertx/ext/mongo/FindOptions.java

```diff
@@ -45,7 +45,7 @@ public FindOptions() {
     this.limit = DEFAULT_LIMIT;
     this.skip = DEFAULT_SKIP;
     this.batchSize = DEFAULT_BATCH_SIZE;
-    this.hint = "";
+    this.hint = null;
```

Comment:
This is a breaking change.

Reply:
Shall I document it in the wiki and/or move it to another PR (like above's issue with `collation` vs `collationOptions`)?